coalib/parsing/FilterHelper.py

```diff
@@ -7,6 +7,22 @@ def is_valid_filter(filter):
 
 
 def apply_filter(filter_name, filter_args, all_bears=None):
+    """
+    Returns bears after filtering based on ``filter_args``. It returns
+    all bears if nothing is present in ``filter_args``.
```

Comment:
No need to specify this explicitly. Its obvious that if there's no filter, we have to return all the bears.